docs/components/Footer.js

```diff
@@ -4,40 +4,80 @@ import { Box, Flex, Icon, Link, Text } from 'gestalt';
 import trackButtonClick from './buttons/trackButtonClick.js';
 
 export default function Footer(): Node {
+  const links = [
+    { title: "What's new", url: 'https://gestalt.pinterest.systems/home' },
+    { title: 'About us', url: 'https://gestalt.pinterest.systems/about_us' },
+    { title: 'Careers', url: 'https://www.pinterestcareers.com/', external: true },
+    { title: 'Code sandbox', url: 'https://codesandbox.io/s/k5plvp9v8v', external: true },
+    { title: 'Github', url: 'https://github.com/pinterest/gestalt', external: true },
+  ];
+
   return (
-    <Box color="lightGray" padding={4} mdPadding={6} lgPadding={8} role="contentinfo">
-      <Box paddingX={2} paddingY={1} display="flex" direction="column" mdDirection="row">
+    <Box
+      dangerouslySetInlineStyle={{
+        __style: {
+          backgroundColor: 'var(--color-orange-firetini-0)',
+        },
+      }}
+      padding={4}
+      role="contentinfo"
+    >
+      <Flex direction="row" alignItems="center" wrap>
         <Box column={12} mdColumn={3} padding={2}>
           <Flex alignItems="center" gap={2}>
             <Box aria-hidden>
-              <Icon icon="pinterest" color="brandPrimary" size={24} accessibilityLabel="" />
+              <Icon icon="pinterest" color="default" size={24} accessibilityLabel="" />
             </Box>
-            <Text color="default" weight="bold">
+            <Text size="100">
               <Link
                 accessibilityLabel="Visit Pinterest.com"
                 href="https://www.pinterest.com"
-                onClick={() => trackButtonClick('Pinterest logo')}
+                onClick={() => trackButtonClick('Pinterest copyright')}
               >
-                Pinterest
+                &copy; {new Date().getFullYear()} Pinterest
```

Comment:
ahh thank you so much for doing this the smart way 💯 😄 

(I wish all websites did)

Reply:
hah same!